### ISSUE TITLE
Accessibility Research

### ISSUE THREAD

**Issue by jrsalata** [jrsalata/toolery#37]
This issue is a part of an AI-guided research project for a software testing class.

We will be working on performing a review on each of the main sections of the user interface and identify areas in which accessibility might be a concern. This includes color contrast, screen reader compatibility, and overall usability. Those that use various accessibility features should be able to use the app as easily as possible.

We will then work on making non-functional tests to ensure that future iterations remain accessible based on the changes made. These tests should be specific to each feature/page set.
